app/mailers/action_notifications_mailer.rb

```diff
@@ -44,6 +44,18 @@ def ready_for_approver_review(assignment)
     mail(to: recipient.email)
   end
 
+  def account_not_active(record)
```

Comment:
It sounds like this would be more appropriate in the `devise_mailer.rb` file ... this `action_notifier_mailer.rb` is more about notifying the user about case-related actions.